### DISCUSSION TITLE
inertiajs-starter

### DISCUSSION THREAD

**Discussion by domhaas** [lunarphp/lunar#1724] [Roadmap]
As I've read https://twitter.com/lunarphp/status/1768715489289621853 are there any news on an inertiajs-starterkit? Would be quiet cool.
Thank you in advance.

**Comment by glennjacobs** [comment]
It's in our plans, but something we'll release after v1.0 is out.

**Reply by domhaas** [reply]
Thank's @glennjacobs for the quick reply!
Do you think there are any issues/blockers to use inertiajs instead of livewire for the storefront in the current version of lunar?

**Reply by glennjacobs** [reply]
Zero. It's what we use and put a large site live on Tuesday using it. I would recommend it in fact for larger sites.

**Reply by domhaas** [reply]
good to hear. thank's a lot!
